client/components/premium-global-styles-upgrade-modal/index.tsx

```diff
@@ -35,6 +38,13 @@ export default function PremiumGlobalStylesUpgradeModal( {
 		translate( 'Ad-free experience' ),
 		translate( 'Earn with WordAds' ),
 	];
+	const personalFeatures = [
+		<strong>{ translate( 'Free domain for one year' ) }</strong>,
+		translate( 'Style customization' ),
+		translate( 'Live chat support' ),
```

Comment:
The Personal plan does not have live chat support. Maybe we can highlight the gated/premium content feature?

Reply:
Done